extensions/flyway/deployment/src/main/java/io/quarkus/flyway/FlywayProcessor.java

```diff
@@ -220,28 +215,19 @@ ServiceStartBuildItem createBeansAndStartActions(FlywayRecorder recorder,
                 continue;
             }
 
-            Enumeration<URL> migrations = Thread.currentThread().getContextClassLoader().getResources(location);
-            while (migrations.hasMoreElements()) {
-                URL path = migrations.nextElement();
-                LOGGER.infov("Adding application migrations in path ''{0}'' using protocol ''{1}''", path.getPath(),
```

Comment:
we lost this AFAICS but I think it was too verbose before so I agree with the change